iphone/Maps/UI/PlacePage/PlacePageViewController.swift

```diff
@@ -84,6 +84,11 @@ final class PlacePageScrollView: UIScrollView {
 
   override func viewDidLayoutSubviews() {
     super.viewDidLayoutSubviews()
+    if #unavailable(iOS 13.0), previousTraitCollection == nil {
```

Comment:
This doesn't compile with XCode 13...

Reply:
Looks like this approach should be used instead: https://github.com/ProtonMail/ios-mail/blob/bad1e950430f3ae1d893dc8e4ad5f4833ecfbfe2/ProtonMail/ProtonMail/AppDelegate.swift#L226